src/calls/sc.js

```diff
@@ -178,6 +178,24 @@ export const addScType = async (
   }
 };
 
-export const addScStatus = () => {
-  // TODO
+export const setStatus = async (scId, status, error) => {
```

Comment:
I would set another naming for this function. Functions beginning with 'set' are used in React components, therefore it might cause confusion. My suggestion is simply 'addStatus'

Reply:
changed ;)